docs/content/guides/authn/oidc_first_party_auth.adoc

```diff
@@ -27,22 +27,21 @@ There are also some further endpoints, which you'll learn about during the setup
 
 Technically, the setup includes:
 
-* https://hub.docker.com/r/traefik/whoami/[traefik/whoami] - A service that echoes back everything it receives, simulating our main service with endpoints mentioned above.
+* https://hub.docker.com/r/traefik/whoami/[traefik/whoami] - A service that echoes back everything it receives, simulating our main service on the endpoint defined above.
```

Comment:
I'm concerned that some information might be lost here. Multiple endpoints were defined, so it might be clearer to say: "A service that echoes back everything it receives, simulating our main service, which exposes the endpoints defined above."